bingads_downloader/downloader.py

```diff
@@ -210,8 +210,11 @@ def authenticate_with_oauth(api_client):
             print('No refresh token found. Please run refresh refresh-bingads-api-oauth2-token')
             sys.exit(1)
     except OAuthTokenRequestException as exc:
-        print('Authentication error. Could be necessary to run refresh refresh-bingads-api-oauth2-token')
-        print(exc)
+        print('Authentication error. Could be necessary to run refresh refresh-bingads-api-oauth2-token', file=sys.stderr)
+        print('Ensure that the registered application type is Native-Application', file=sys.stderr)
+        print(f' - client id: {config.oauth2_client_id()}', file=sys.stderr)
```

Comment:
If you are using f-strings `bingads-performance-downloader` will require Python >=3.6. You have to update the documentation as well.

Reply:
Right. I will remove f-strings to make it work on older versions